twitter_radix/twitter_radix/components/container.py

```diff
@@ -0,0 +1,21 @@
+"""A container component."""
+import reflex as rx
+
+def container(*children, **props):
+    """A fixed container based on a 960px grid."""
+    # Enable override of default props.
+    props = (
+        dict(
+            width="100%",
+            max_width="960px",
+            background="white",
+            height="100%",
+            #px=[4, 12],
```

Comment:
can we remove the commented code. there also seems to be an empty line below line 15

Reply:
removed